pmm/aws-staging-start.groovy

```diff
@@ -33,6 +33,10 @@ pipeline {
             choices: ['yes', 'no'],
             description: 'Enable Testing Repo?',
             name: 'ENABLE_TESTING_REPO')
+        choice(
+            choices: ['no', 'yes'],
+            description: 'Enable Release Candidate Repo?',
```

Comment:
what do you mean by "Release Candidate Repo"
We have laboratory, experimental, testing and release repos. Let's use standard names

Reply:
It is confusing because we are building pmm-server `dev-latest` on `Laboratory Repo` but pmm2-client `dev-latest` is being build on `testing` repo. 
we already use the term `testing` repo as one option for testing dev-latest, I thought to avoid more confusion, we just use the term rc repo, I will change to actuals